docs: Add information about how to manual query if there are more than 1 SO (#627)
kedacore/keda-docs: content/docs/2.4/operate/metrics-server.md

You can recover the metric names from a ScaledObject using `kubectl`:
```bash
 kubectl get scaledobject SCALEDOBJECT_NAME -n NAMESPACE -o jsonpath={.status.externalMetricNames}
```

> ## How to get metric if there are more than 1 ScaledObject with same metric name

Reviewer: ```suggestion
## How to get metric when multiple ScaledObjects have the same metric name
```

Author: > LGTM, but added some suggestions for all versions

Nice, I will update all of them, thanks for the text improvements :)

> KEDA will try to select the proper `ScaledObject` for your metric and it should be only one. In case of having more than 1 `ScaledObject` in the same namespace with the same metric name, an error like this will be thrown:

Reviewer: ```suggestion
KEDA will try to select the proper `ScaledObject` for your metric and there should only be one. In case of having multiple `ScaledObject`s in the same namespace with the same metric name, an error like this will be thrown:
```

```
Error from server: exactly one ScaledObject should match label
```

> In this case, you should add in the query string the labelSelector to match the proper `ScaledObject` (in url format). The needed selector is `scaledobject.keda.sh/name: {ScaledObjectName}`. You also could get it doing:

Reviewer: ```suggestion
In this case, you should add in the query string the `labelSelector` to match the proper `ScaledObject` (in url format). The needed selector is `scaledobject.keda.sh/name: {ScaledObjectName}`. You can achieve this as following:
```

```bash
 kubectl get scaledobject SCALEDOBJECT_NAME -n NAMESPACE -o jsonpath={.metadata.labels}
```

> Having the selector, you only need to add it in the query string:

Reviewer: ```suggestion
Once you have the selector, you have to add it to your query string as following:
```
